addons/account/wizard/account_payment_register.py

```diff
@@ -191,7 +191,11 @@ def _get_line_batch_key(self, line):
 
         partner_bank_account = self.env['res.partner.bank']
         if move.is_invoice(include_receipts=True):
-            partner_bank_account = move.partner_bank_id._origin
+            if move.move_type == 'in_refund':
```

Comment:
I'm seeing this now:
From my understanding, you fix is for the credit note sent (`out_refund`) on invoices (`out_invoice`).
But here, you are using the credit note received (`in_refund`) on bills (`in_invoice`)